vignettes/getting-started.Rmd

```diff
@@ -87,6 +86,7 @@ gimap_dataset <-  gimap::setup_data(counts = example_counts,
 
 You'll notice that this set up gives us a list of formatted data. This contains the original counts we gave `setup_data()` function but also normalized counts, and the total counts per sample. 
 
+
```

Comment:
`gimap_dataset` is a list with 5 elements inside. Let's list out these elements and give a 1-sentence description:

For ex,

- `raw_counts`: Raw counts
- `counts_per_sample`: Counts per sample
- `coverage`: 
- `transformed_data`: Transformed dataset
- `metadata`: Metadata

Reply:
I like this! Can you add this in @howardbaek ?

Reply:
Can @danieljgroso or anyone else give more details on these elements?

Reply:
I think we ended up covering this in the meeting but let me know if there's any info you need.